windows/INSTALL.md

```diff
@@ -121,7 +120,8 @@ for their processors. It isn't free, but you can get [Community Licensing for In
          (kaldi)/tools/openfst$ cd ../../windows
          (kaldi)/windows $ pwd
 
-11. Modify the file `variables.props` to reflect
+11. Copy `variables.props.dev` to `variables.props`.
```

Comment:
@jtrmal, did you see this PR?

On Tue, Mar 15, 2016 at 7:30 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In windows/INSTALL.md
> <https://github.com/kaldi-asr/kaldi/pull/582#discussion_r56259379>:
>
> > @@ -121,7 +120,8 @@ for their processors. It isn't free, but you can get [Community Licensing for In
> >           (kaldi)/tools/openfst$ cd ../../windows
> >           (kaldi)/windows $ pwd
> >
> > -11. Modify the file `variables.props` to reflect
> > +11. Copy `variables.props.dev` to `variables.props`.
>
> Could you please add variables.props and kaldiwin.props to .gitignore?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/582/files/f939f2d6463f99b0eb0943e855e9d414f7d11de1#r56259379>
>